README.md

```diff
@@ -178,6 +234,9 @@ of the application:
 
 | name                       | description                                                          | typical value              |
 | -------------------------- | -------------------------------------------------------------------- | -------------------------- |
-| `RAILS_RELATIVE_URL_ROOT`  | The path from the server root to the application                     | `/app/ppd`                 |
+| `APPLICATION_ROOT`         | The path from the server root to the application                     | `/app/ppd`                 |
```

Comment:
NO. This shouldn't be an environment variable unless the image has no knowledge of the path from build time. I VERY much suspect this is not the case.

What does "Typical Value" mean?